### DISCUSSION TITLE
nat64  must be supported?

### DISCUSSION THREAD

**Discussion by wallaceL** [openthread/openthread#8349] [Q&A] [answered]
The spec says the BR must support IPV4 and IPV6 dual-stack capabilities.  In otbr/openthread source the nat64 feature should not be configurable, and should be mandatory.  Is it right?

**Comment by jwhui** [comment, marked as answer]
The Thread Group deferred certification testing of NAT64 until 2023. As a result, the OpenThread implementation leaves it as an optional feature for now.

**Reply by lecontr2** [reply]
May I ask where I may find out if NAT64 is currently implemented in OTBRs that are currently released and purchasable from stores (Google's Nest Hub (2nd Gen), Apple HomePod Mini, Apple 4K TV, Samsung Aeotec v3, Eero Router Pro 6E)?

**Reply by jwhui** [reply]
I'm not aware of any listing that provides this information.

**Reply by lecontr2** [reply]
Ah I see... What would be the best way to check if the NAT64 feature is enabled on these hubs? Request the netdata and check if the NAT64 prefix is returned?

**Reply by jwhui** [reply]
Yes, the NAT64 prefix, when it exists, will be included in Thread Network Data as an External Route with the NAT64 flag set.

https://github.com/openthread/openthread/blob/main/src/cli/README_NETDATA.md#publish-route-prefix-snprf

**Reply by lecontr2** [reply]
I see, thank you for the clarification!
